### PR TITLE
Fix sed script and add echo with newline

### PR DESCRIPTION
**Pull request by ahanafy**
Removed apostrophes from sed line, and added a newline when appending new team directory to kustomize.yaml file

### CODE REVIEW

File: scripts/create-team.sh [hunk 1]
```diff
@@ -15,9 +15,9 @@ cp -r "${REPO_ROOT}/cluster/${TEMPLATE}/." ${TEAM_DIR}
 
 for f in "${TEAM_DIR}*.yaml"
 do
- sed -i '' "s/$TEMPLATE/$TEAM_NAME/g" ${f}
+ sed -i "s/$TEMPLATE/$TEAM_NAME/g" ${f}
```
Comment (stefanprodan) [inline scripts/create-team.sh]: Have you tested this on macOS?

Comment (ahanafy) [reply scripts/create-team.sh]: I didn't test on OSX but i noticed there were some issues with sed compatibility.
I submitted a new change with a version of the sed command that is more compatible with OSX 10.9+ and GNU.
I don't have a mac, but if you do, could you confirm that the sed command works?

Comment (stefanprodan) [reply scripts/create-team.sh]: It doesn't work:
```
tree cluster/team2
cluster/team2
├── flux-patch.yaml
├── flux-patch.yaml-e
├── kubeconfig.yaml
├── kubeconfig.yaml-e
├── kustomization.yaml
├── kustomization.yaml-e
├── namespace.yaml
├── namespace.yaml-e
├── psp.yaml
├── psp.yaml-e
├── rbac.yaml
└── rbac.yaml-e
```

Comment (stefanprodan) [reply scripts/create-team.sh]: You could switch the commands based on `uname` and use the old command on Darwin.

Comment (ahanafy) [reply scripts/create-team.sh]: Did you test on 10.6 or 10.9+? The `uname` trick is what was listed in the stackoverflow thread, but what is the difference in `uname` output between 10.6 vs 10.9+? Is there some way to determine if 10.6 vs 10.9? I figured at least supporting 10.9+ was a good start. I think 10.6 is around 10 years old?

**Review by ahanafy — COMMENTED** [review]
This is the version that should work on 10.9+ and GNU

**Comment by stefanprodan** [comment]
Closing in favour of #6

**Pull request closed without merging**